labrad/types.py

```diff
@@ -394,6 +389,28 @@ def _flatten_to(obj, types, endianness):
     return s, t
 
 
+def __flatten__(t, obj, endianness):
+    """Flatten an object to the specific given type.
```

Comment:
The obvious would be _flatten

On Thu, Mar 3, 2016 at 9:43 AM, Matthew Neeley <notifications@github.com>
wrote:

> In labrad/types.py
> <https://github.com/labrad/pylabrad/pull/223#discussion_r54917303>:
>
> > @@ -394,6 +389,28 @@ def _flatten_to(obj, types, endianness):
> >      return s, t
> >
> >
> > +def __flatten__(t, obj, endianness):
> > +    """Flatten an object to the specific given type.
>
> I agree on all fronts. I've added a method flatten on LRType which
> handles the case where the data is already flattened, and otherwise
> delegates real flattening work to the existing __flatten__ method (which
> we might want to rename).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/labrad/pylabrad/pull/223/files#r54917303>.
>